codegen/src/main/resources/cordapp/cordapp_gradle.mustache

```diff
@@ -4,7 +4,7 @@ buildscript {
         corda_release_group = 'net.corda'
         corda_release_version = '4.1'
         corda_gradle_plugins_version = '4.0.45'
-        kotlin_version = '1.2.71'
+        kotlin_version = '1.3.50'
```

Comment:
Not sure if we can upgrade the Kotlin version, Corda only supports 1.2.71 so the generated CorDapp should keep this version no?